sidecar.Dockerfile

```diff
@@ -32,6 +32,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends fuse &&\
   apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 RUN echo "allow_root" >> /etc/fuse.conf
 
+# RUN echo "user_allow_other" >> /etc/fuse.conf
```

Comment:
Does this seem to do the trick? (of giving permission to the sidecar container to /dev/fuse in docker-for-desktop context)

Reply:
no, it does not.  still unclear about the linkage between https://github.com/jacobsa/fuse and [`fusermount`](http://man7.org/linux/man-pages/man8/mount.fuse.8.html) if any.  i have read much of the internal message-passing dataflow in jacobsa/fuse, yet the filesystem io to `/dev/fuse` is still unread.

it could be that some options need to be passed to jacobsa/fuse rather than set in `/etc/fuse.conf`.

overall, i am supposing that maintaining a single image with a non-root user rather than shipping two separate sidecar images is an option.  indeed, perhaps even an easier option.